src/Fable.Cli/Fable.Cli.fsproj

```diff
@@ -48,6 +48,7 @@
     <Compile Include="FileWatchers.fsi" />
     <Compile Include="FileWatchers.fs" />
     <Compile Include="Pipeline.fs" />
+    <Compile Include="MSBuildCrackerResolver.fs" />
```

Comment:
I just saw that there is a typo in the name of the File

Reply:
What is the typo? I fail to see it 😅

Reply:
I placed the comment on the wrong line 😅